src/models/IError.ts

```diff
@@ -0,0 +1,3 @@
+export default interface IError extends Error {
+    errorCode?: number;
```

Comment:
Forgot to add these message & name properties?

Reply:
IError extends Error, message and name properties are included in Error interface

Reply:
Ah I see